### ISSUE TITLE
Docx files generated by the library(v1.4.7) are corrupt 

### ISSUE THREAD

**Issue by grigala** [verronpro/docx-stamper#91]
When generating a `.docx` file from a template, the Word is not able to open it.

It's weird that the LibreOffice or even Preview on macOS is able to open it just fine. 
I'm suspecting some sort of metadata is missing from the file that Microsoft Word is not happy about. 

I know there are new versions of the docx-stamper out there, but those have the Jakarta EE 9 migration in it. This migration prevents our project to use those new versions because other dependencies in our codebase use old version of JAXB that effectively causes clash between `javax.xml.bind.JAXBElement` and `jakarta.xml.bind.JAXBElement`. 
![image](https://user-images.githubusercontent.com/2084794/196375419-3012ab2f-f26b-4232-8ea0-193cd14d2e1a.png)
I was not able to find a solution how to mitigate this issue, hence usage of 1.4.7 version. 

I would be happy to help if I can, but unfortunately I'm not fully familiar with the codebase. 

Nothing particularly interesting is reported in the logs(to my eyes, there are a couple of warnings but nothing else):
```[http-nio-8080-exec-2] INFO  [user=] org.docx4j.jaxb.Context - java.vendor=IBM Corporation
[http-nio-8080-exec-2] INFO  [user=] org.docx4j.jaxb.Context - java.version=17.0.3
[http-nio-8080-exec-2] WARN  [user=] o.d.jaxb.NamespacePrefixMapperUtils - com.sun.xml.internal.bind.marshaller.NamespacePrefixMapper .. trying RI.
[http-nio-8080-exec-2] INFO  [user=] o.d.jaxb.NamespacePrefixMapperUtils - Using NamespacePrefixMapper, which is suitable for the JAXB RI
[http-nio-8080-exec-2] INFO  [user=] org.docx4j.jaxb.Context - MOXy JAXB implementation is in use!
[http-nio-8080-exec-1] WARN  [user=] org.docx4j.utils.ResourceUtils - Couldn't get resource: docx4j.properties
[http-nio-8080-exec-1] WARN  [user=] org.docx4j.Docx4jProperties - Couldn't find/read docx4j.properties; docx4j.properties not found via classloader.
[http-nio-8080-exec-2] WARN  [user=] org.docx4j.utils.ResourceUtils - Couldn't get resource: docx4j.properties
[http-nio-8080-exec-2] WARN  [user=] org.docx4j.Docx4jProperties - Couldn't find/read docx4j.properties; docx4j.properties not found via classloader.
[http-nio-8080-exec-2] WARN  [user=] org.docx4j.XmlUtils - default SAXParserFactory property : null
        Please consider using Xerces.
[http-nio-8080-exec-2] INFO  [user=] org.docx4j.XmlUtils - actual: com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl
[http-nio-8080-exec-2] WARN  [user=] org.docx4j.XmlUtils - default DocumentBuilderFactory property: null
        Please consider using Xerces.
[http-nio-8080-exec-2] INFO  [user=] org.docx4j.XmlUtils - actual: com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderFactoryImpl
[http-nio-8080-exec-2] INFO  [user=] o.d.o.contenttype.ContentTypeManager - Detected WordProcessingML package
[http-nio-8080-exec-1] INFO  [user=] o.d.o.contenttype.ContentTypeManager - Detected WordProcessingML package
[http-nio-8080-exec-2] INFO  [user=] o.d.o.contenttype.ContentTypeManager - Detected WordProcessingML package
[http-nio-8080-exec-1] INFO  [user=] o.d.o.contenttype.ContentTypeManager - Detected WordProcessingML package
[http-nio-8080-exec-2] INFO  [user=] org.docx4j.openpackaging.io3.Load3 - Instantiated package of type org.docx4j.openpackaging.packages.WordprocessingMLPackage
[http-nio-8080-exec-1] INFO  [user=] org.docx4j.openpackaging.io3.Load3 - Instantiated package of type org.docx4j.openpackaging.packages.WordprocessingMLPackage
[http-nio-8080-exec-2] INFO  [user=] org.docx4j.utils.XPathFactoryUtil - xpath implementation: org.apache.xpath.jaxp.XPathFactoryImpl
[http-nio-8080-exec-1] WARN  [user=] o.d.o.contenttype.ContentTypeManager - DefaultPart used for part '/word/media/image2.svg' of content type 'image/svg+xml'
[http-nio-8080-exec-2] WARN  [user=] o.d.o.contenttype.ContentTypeManager - DefaultPart used for part '/word/media/image2.svg' of content type 'image/svg+xml'
[http-nio-8080-exec-2] WARN  [user=] o.d.o.contenttype.ContentTypeManager - DefaultPart used for part '/word/commentsExtensible.xml' of content type 'application/vnd.openxmlformats-officedocument.wordprocessingml.commentsExtensible+xml'
[http-nio-8080-exec-1] WARN  [user=] o.d.o.contenttype.ContentTypeManager - DefaultPart used for part '/word/commentsExtensible.xml' of content type 'application/vnd.openxmlformats-officedocument.wordprocessingml.commentsExtensible+xml'
[http-nio-8080-exec-2] WARN  [user=] o.d.o.contenttype.ContentTypeManager - DefaultPart used for part '/word/commentsIds.xml' of content type 'application/vnd.openxmlformats-officedocument.wordprocessingml.commentsIds+xml'
[http-nio-8080-exec-1] WARN  [user=] o.d.o.contenttype.ContentTypeManager - DefaultPart used for part '/word/commentsIds.xml' of content type 'application/vnd.openxmlformats-officedocument.wordprocessingml.commentsIds+xml'
[http-nio-8080-exec-1] INFO  [user=] org.docx4j.openpackaging.io3.Load3 - package read;  elapsed time: 167 ms
[http-nio-8080-exec-2] INFO  [user=] org.docx4j.openpackaging.io3.Load3 - package read;  elapsed time: 167 ms
[http-nio-8080-exec-1] WARN  [user=] o.d.jaxb.NamespacePrefixMapperUtils - com.sun.xml.internal.bind.marshaller.NamespacePrefixMapper .. trying RI.
[http-nio-8080-exec-2] WARN  [user=] o.d.jaxb.NamespacePrefixMapperUtils - com.sun.xml.internal.bind.marshaller.NamespacePrefixMapper .. trying RI.
[http-nio-8080-exec-1] INFO  [user=] o.d.jaxb.NamespacePrefixMapperUtils - Using NamespacePrefixMapperRelationshipsPart, which is suitable for the JAXB RI
[http-nio-8080-exec-2] INFO  [user=] o.d.jaxb.NamespacePrefixMapperUtils - Using NamespacePrefixMapperRelationshipsPart, which is suitable for the JAXB RI
```

**Screenshots**
<img width="372" alt="image" src="https://user-images.githubusercontent.com/2084794/196270404-d3f1dfdd-5a22-4725-92cf-7b6e65946ee6.png">

<img width="530" alt="image" src="https://user-images.githubusercontent.com/2084794/196275785-676319e1-7dbe-4913-8e2c-581904184128.png">

**Desktop (please complete the following information):**
 - OS: macOS Monterey & Windows 10
 - Java version: 17
 - docx4j version: 6.1.2
 - docx-stamper version: 1.4.7 (Can't go higher than that due to Jakarta EE 9)

**Comment by grigala** [comment]
Ok I just discovered that docx4j 6.1.2 still using `javax.xml.bind` packages, so I'm guessing if I drop `org.docx4j.docx4j:6.1.2` and use `org.docx4j:docx4j-core:11.4.8` I wouldn't get `java.lang.ClassCastException: javax.xml.bind.JAXBElement incompatible with jakarta.xml.bind.JAXBElement` anymore with 1.5.0. 

Yes this worked, and the files are not corrupt anymore with the newest versions. 

Closing this.
